Purchases/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.12.0-SNAPSHOT</string>
+	<string>3.12.0</string>
```

Comment:
`CFBundleShortVersionString` can only have numbers and dots, otherwise the app gets rejected. So the stuff int the `.plist` files doesn't have the `-beta1` suffix